File: 283-move-zeroes/283-move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        
        count=0
        
        for x in nums:
            
            if(x!=0):
                
                nums[count]=x
                count+=1
   
        for i in range(count,len(nums)):
            
            nums[i]=0

File: 283-move-zeroes/283-move-zeroes.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        
        int count = 0;
        for(auto i = nums.begin(); i != nums.end(); i++){
            
            if(*i != 0){
                nums[count] = *i;
                count++;
            }
        }
        
        for (int i = count; i< nums.size(); i++){
            
            nums[i] = 0;
        }
    }
};